scripts/fetch.R

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/Rscript
```

Comment:
Looks like this got accidentally deleted in last PR. Needed to run this as a command line script.